src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -336,6 +336,31 @@ internal Authority GetAuthority(IAccount account)
             return await handler.RunAsync(CancellationToken.None).ConfigureAwait(false);
         }
 
+        internal async Task<AuthenticationResult> ExchangeRefreshTokenAsync(string userProvidedRefreshToken)
+        {
+            var reqParams = new AuthenticationRequestParameters
+            {
+                SliceParameters = SliceParameters,
+                Authority = Instance.Authority.CreateAuthority(ServiceBundle, Authority, false),
+                ClientId = ClientId,
+                TokenCache = UserTokenCache,
+                Scope = new SortedSet<string>(),
```

Comment:
Why do we request 0 scopes?

Reply:
This has been changed to send the clientid/.default as the scope if the user passes nothing in. The goal of this feature is to allow a user to provide a V1 refresh token and exchange it for a V2 refresh token. The user will then be able to make silent calls with the cached V2 refresh token.

Reply:
WE can follow up in our 1:1